### PR TITLE
fixed a few typos and wrapped some long sentences

### PR DESCRIPTION
**Pull request by cshoben**
- fixed some typos

- changed some words to match style

                  made BedToFasta description match singleBedToFasta

- wrapped some long sentences to make reading easier

### CODE REVIEW

**Review by craiglowe — DISMISSED** [review]
Looks good!

File: convert/convert.go [hunk 1]
```diff
@@ -169,7 +172,8 @@ func getWigChromIndex(s string, wigSlice []wig.Wig) int {
 	return -1
 }
 
-//PairwiseFaToVcf takes in a pairwise multiFa alignment and writes Vcf entries for segregating sites with the first entry as the reference and the second fasta entry as the alt allele.
+//PairwiseFaToVcf takes in a pairwise multiFa alignment and writes Vcf entries for segregating sites with the first
+// entry as the reference and the second fasta entry as the alt allele.
```
Comment (dasnellings) [inline convert/convert.go:176]: Very minor, but this line has a space before the text whereas the line above does not. May be something to update in future godoc-related PRs.

**Review by dasnellings — DISMISSED** [review]

File: convert/convert.go (hunk 1 above)
Comment (cshoben) [reply convert/convert.go]: I'll just update it now. If you can re-approve that'd be great!

**Review by craiglowe — APPROVED** [review]
Great!

**Review by RavenYL — APPROVED** [review]

**Pull request merged**